crates/sui-indexer/migrations/pg/2024-09-26-143533_tx_digests_to_checkpoint_contents/up.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE checkpoints DROP COLUMN tx_digests;
+ALTER TABLE checkpoints ADD COLUMN checkpoint_contents BYTEA;
```

Comment:
This data already lives in the new checkpoints table so we can likely just follow through dropping the tx_digests column and omit adding this one